Makefile

```diff
@@ -1,4 +1,5 @@
 DC := $(shell which docker-compose)
+WEBAPP_HOST ?= localhost:8000
```

Comment:
Make this ``"http://localhost:8000"`` instead.

Reply:
Also, can you add a comment to explain why you want to able to set this variable from the environment?

Reply:
By this I mean, can you explain the use case that requires you to use a variable here? Like, something about Windows needing to expose the service on a different URL.